src/fastpath/fastpath.c

```diff
@@ -175,7 +177,7 @@ void NORETURN fastpath_call(word_t cptr, word_t msgInfo)
 #endif
 
     /* Dequeue the destination. */
-    endpoint_ptr_set_epQueue_head_np(ep_ptr, TCB_REF(dest->tcbEPNext));
```

Comment:
Why can't you change `TCB_REF` instead?

Reply:
I don't mind changing to `TCB_REF` but I thought `TCB_PTR` is better when it comes to naming/intentionality; like this is now really a pointer and not a REF or INTEGER. I was under the impression that _REF macros as always integers for some reason and left them as is